runhouse/servers/cluster_servlet.py

```diff
@@ -664,11 +664,11 @@ def status(self, send_to_den: bool = False):
     ##############################################
     # Save cluster logs to Den
     ##############################################
-    def _get_logs(self):
+    def _get_logs(self, join_by: str = " "):
```

Comment:
in what scenarios would the `join_by` be different?

Reply:
Added it to support getting logs, can be removed now, since we are waiting for Rohin's implementation.